upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1410,6 +1410,26 @@ There you can find links to upgrade notes for other versions too.
 - javascript assets are managed by webpack and npm ([#1545](https://github.com/shopsys/shopsys/pull/1545), [#1645](https://github.com/shopsys/shopsys/pull/1645))
     - please read [upgrade instruction for webpack](./upgrade-instruction-for-webpack.md)
 
+- css and other assets are managed by webpack ([#1725](https://github.com/shopsys/shopsys/pull/1725))
+    - see #project-base-diff to update your project
+    - move content from `src/resources/styles/front` to `assets/styles/frontend`
+    - move content from `src/resources/styles/admin` to `assets/styles/admin`
+    - move content from `src/resources/svg` to `assets/styles/frontend/svg`
```

Comment:
Come on! :disappointed: 
Should not it be like this?
```suggestion
    - move content from `src/resources/svg/front` to `assets/public/frontend/svg`
```

Reply:
damn, did not notice it is merged already...I will create an issue...

Reply:
You are right. Thx, I will fix it.

Reply:
thank you @pesektomas https://github.com/shopsys/shopsys/issues/1778 :wink:

Reply:
Yea, I am looking on it, thank you too